pkg/virt-api/rest/authorizer.go

```diff
@@ -138,34 +140,36 @@ func (a *authorizor) generateAccessReview(req *restful.Request) (*authv1.Subject
 		return nil, err
 	}
 
-	r := &authv1.SubjectAccessReview{}
-	r.Spec = authv1.SubjectAccessReviewSpec{
-		User:   userName,
-		Groups: userGroups,
-		Extra:  a.getUserExtras(req.Request.Header),
+	return &authv1.SubjectAccessReview{
+		Spec: authv1.SubjectAccessReviewSpec{
+			User:   userName,
+			Groups: userGroups,
+			Extra:  a.getUserExtras(req.Request.Header),
+		},
+	}, nil
+}
+
+func (a *authorizor) CreateSubjectAccessReview(sar *authv1.SubjectAccessReview) (*authv1.SubjectAccessReview, error) {
```

Comment:
I think this method can be removed. It only calls the client inside `authorizor`, which is passed to method `NewAuthorizorFromClient()`. A potential caller of this method can instead call the client directly.